pkg/helper/versions/helpers.go

```diff
@@ -18,7 +18,7 @@ const (
 )
 
 func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP bool, filterHostedCP bool,
```

Comment:
@chenz4027 You're currently not meeting our requirements on adding test coverage for changes to ROSA which is why the build on your PR is failing.

From your changes, this looks like the easiest function to write a test for. If you can do that, your build should pass.

Reply:
Thanks @robpblake , I will add a test for it 👍